libs/tokens/src/const/tokensList.json

```diff
@@ -10,6 +10,11 @@
       "enabledByDefault": true,
       "source": "https://files.cow.fi/tokens/CoinGecko.json"
     },
+    {
+      "priority": 13,
+      "enabledByDefault": true,
+      "source": "https://dpaste.com/3SCBYM7GS.txt"
```

Comment:
Should we add similar list to Gnosis chain? Or all "lucky" tokens already included in the default lists?